GOVERNANCE.md

```diff
@@ -0,0 +1,119 @@
+the collection of positive/negative votes associated with an identity may be used.
+Examples of this include voting through "thumbs up/down" emojis or with "+1"
+comments in issues.
+
+Maintainer consensus shall be reached in the following circumstances:
+
+- Having a majority of favorable maintainer votes at the end of the proposal
+discussion term which may not be longer than thirty (30 ) days. The number of
+favorable opinions may never be less than two(2).
+
+- Achieving a favorable vote of more than 50% of the total active maintainers
+at any time during the discussion period.
+
+- In all cases, when the number of organizations with active maintainers in
+the project exceeds four (4), the total vote count shall include maintainers
+from at least two different organizations.
+
+- The one exception to reaching maintainer consensus is when voting for
+proposals that modify the governance model (see below).
+
+Note that when qualified, the proponents may add their favorable vote to count towards consensus but needs to be explicitly recorded in the voting mechanism.
+
+## Review Criteria for OpenVEX Repositories
+
+Any changes intended to be merged in the OpenVEX repositories shall meet the following minimal criteria:
+
+- Commits must be signed by the author
+- Commits must be signed off
+- Pull requests must be approved by at least one of the project's maintainers
+
+Any repository under the OpenVEX organization may impose additional requirements
+to approve pull requests as long as these minimal requirements are met.
+
+## Revisions to the Governance Model
+
+The project's governance Model shall be revisited every six months to address
+the needs of the community as the project recognizes that communities need to
+steer themselves according to their size, members, and other factors that shape
+their complexity. 
+
+At any point, an OpenVEX Enhancement Proposal may be opened to redefine the
+project's governance. To be accepted, governing model proposals shall be
+approved by a qualified majority consisting of a minimum of 66% favorable votes
+of all active maintainers.
+
+## Reaching Maintainer Status
+
+Any community member can be considered as a candidate for maintainer status
+under the following conditions:
+
+- Any community member may self-nominate as a maintainer candidate after
+actively contributing to OpenVEX constantly for six months or more.
+- A sponsoring committee of maintainers that meets the qualifying criteria may
+nominate any community member as a candidate.
+
+### Sponsoring Committees
+
+To nominate a community member as a maintainer candidate, a group of
+maintainers may file a nomination. The commute shall meet 5he following criteria
+to be qualified to file the nomination:
+
+- The number of members in the committee shall not be less than two (2).
+- Whenever the number of organizations with maintainers in the project is more
+than three(3), committee members shall be from different affiliations.
```

Comment:
```suggestion
than three (3), committee members shall be from different affiliations.
```

Also, does "affiliations" mean "organizations"? (Wondering if it'd be more clear to say "organizations" in that case)